hello_phoenix/firmware/config/target.exs

```diff
@@ -14,6 +14,11 @@ config :ui, UiWeb.Endpoint,
   # Nerves root filesystem is read-only, so disable the code reloader
   code_reloader: false
 
+config :ui, Ui.Repo,
+  database: "/data/ui.db",
```

Comment:
```suggestion
  database: "/data/hello_phoenix_ui.db",
```

I'd actually love for this to be `/data/hello_phoenix/ui.db`, but that requires a exqlite fix to create the parent directory. The reason I like this is that SQLite creates temp files and it would be nice if they didn't pollute the root of `/data`.

I don't know how quick that change could get in, so I'm happy with a rename to something other than `ui.db` in the root `/data` directory for now.

Reply:
@ConnorRigby `exqlite` creates missing parent directories now thanks to Jon. If you update deps, this will "just work". Could you update the PR to move the database into the `/data/hello_phoenix` directory?

I think that that's the only change to this PR.